docs/resources/Channel.md

```diff
@@ -518,20 +522,38 @@ Information about the call in a private channel.
 
 ### Message Reference Object
 
-###### Message Reference Structure
+#### Message Reference Structure
 
 | Field               | Type      | Description                                                                                                                             |
 |---------------------|-----------|-----------------------------------------------------------------------------------------------------------------------------------------|
+| type? \*            | integer   | [type of reference](#DOCS_RESOURCES_CHANNEL/message-reference-types).                                                                   |
 | message_id?         | snowflake | id of the originating message                                                                                                           |
-| channel_id? \*      | snowflake | id of the originating message's channel                                                                                                 |
+| channel_id? \*\*    | snowflake | id of the originating message's channel                                                                                                 |
 | guild_id?           | snowflake | id of the originating message's guild                                                                                                   |
 | fail_if_not_exists? | boolean   | when sending, whether to error if the referenced message doesn't exist instead of sending as a normal (non-reply) message, default true |
 
-\* `channel_id` is optional when creating a reply, but will always be present when receiving an event/response that includes this data model.
+\* If `type` is unset, `DEFAULT` can be assumed in order to match the behaviour before message reference had types.
+In future API versions this will become a required field.
 
-#### Message Types
+\*\* `channel_id` is optional when creating a reply, but will always be present when receiving an event/response that includes this data model.
```

Comment:
If this is required (as per line 583) for forwards now, I think it would be good to put this info somewhere around here.